StateSpecs-Specs.package/SpecOfFailed.class/properties.json

```diff
@@ -1,6 +1,6 @@
 {
 	"commentStamp" : "DenisKudryashov 3/2/2016 17:21",
-	"super" : "Error",
+	"super" : "AssertionFailure",
```

Comment:
Not sure about it. But I am ready to be convinced.
The original reason to be just an Error is to avoid dependency to SUnit. Now I see that AssertionFailure in Kernel-Exceptions. So maybe it is a time to make this change.